app/component/map/popups/DynamicParkingLotsPopup.js

```diff
@@ -84,6 +152,7 @@ class DynamicParkingLotsPopup extends React.Component {
     const desc = (
       <div>
         {this.getCapacity()}
+        {this.props.feature.properties.paid_hours ? this.getOpeningHours() : ''}
         {this.getUrl()}
```

Comment:
I'm still confused. Are we talking about opening or paid hours? Required is opening hours. digitransit and livetile layer should use the same notions.

Reply:
It was left as paid_hours by mistake. Opening hours are displayed.